userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -197,14 +197,22 @@ TEST(interesting_syscalls, names_to_sc_set)
 	"signalfd4",
 	});
 	ASSERT_PPM_SC_CODES_EQ(sc_set_truth, sc_set);
+}
+
+/* This test asserts the behavior of `names_to_sc_set` API when corner cases like `accept/accept4` are involved */
+/// todo: @Andreagit97 revisit this test after new APIs to convert from event_names -> sc_set and sc_set -> event_names.
+TEST(interesting_syscalls, names_sc_set_names_corner_cases)
+{
+	/* INCONSISTENCY: `names_to_sc_set` is converting event names to ppm_sc, but this was not its original scope, the original scope was to convert sc_names -> to sc_set  */
+	std::unordered_set<std::string> event_names{"accept", "execve", "syncfs", "eventfd", "umount", "pipe", "signalfd"};
```

Comment:
We now have umount2 as event name and system call name so old tests should pass? Would prefer not purging umount2 from the tests.

Agreed on revisiting all tests after all new APIs are finalized.

We already put this on
/hold until after 2023-03-15 community call formal agreement, see https://github.com/falcosecurity/falco/issues/2443

plus need to decide which one to merge first and which one needs subsequent rebasing <-> https://github.com/falcosecurity/libs/pull/975

Reply:
now this test is correctly retrieving only PPM_SC_UMOUNT from the `umount` name, I have to purge "PPM_SC_UMOUNT2" from the expected sc_set otherwise the test won't pass :thinking:

Reply:
i've added again `UMOUNT_2` and its associated `PPM_SC_UMOUNT2` and obviously tests are green again :)